### PR TITLE
Fixed the release script to work with M.m.p.o

### LINKED ISSUE #565: Operator will fail if released with new version scheme
When reviewing the required changes for #42, where we'll probably change the versioning scheme to `Major.minor.patch.operator` from the current `Major.minor.operator`, I noticed that the `release` script is fixed to use two parts for the underlying Jaeger version (1.13 vs 1.13.0).

Currently, when trying to release the Operator as 1.13.0.0, the script will probably fail because there are two matches for `([\d]+\.[\d]+)` (1.13 and 0.0).

### PR DESCRIPTION
**Pull request by jpkrohling**
Fixes #565

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
The new regex will work even if we add more components to the Operator version, using the first three parts as the Jaeger version.

File: .travis/release.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ fi
 BASE_BUILD_IMAGE=${BASE_BUILD_IMAGE:-"jaegertracing/jaeger-operator"}
 OPERATOR_VERSION=${OPERATOR_VERSION:-$(git describe --tags)}
 OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d\.]+)")
-JAEGER_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)")
+JAEGER_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+\.[\d]+)" | head -n 1)
```
Comment (objectiser) [inline .travis/release.sh]: Why did you add the `| head -n 1`? Doesn't seem to be required.

Comment (jpkrohling) [reply .travis/release.sh]: When we have four parts, the regex matches two digits plus two digits, generating two matches. The `head -n 1` part will get only the first one.

```
$ echo $OPERATOR_VERSION
1.13.1.2
$ echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)"
1.13
1.2
$ echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)" | head -n 1
1.13
```

The same would happen if we have six parts in the operator version. Very unlikely, so, `head -n` is pretty much unnecessary, but it's not hurting either.

Comment (objectiser) [reply .travis/release.sh]: But the regex matches 3 digits now - it doesn't have any impact, so can be left, but just wasn't sure why had been added.
